src/ps-web-apis.ts

```diff
@@ -174,8 +174,31 @@ export interface WhoamiV1 {
      * @throws error
      */
     getJaId(): string;
+
+    /**
+     * will render the Wonderwall in the given container with the given props and call the callback after main functionality is done
+     *
+     * @param container - The HTML element in which the Wonderwall should be rendered.
+     * The container should be an HTML element.
+     *
+     * @param props - The props that should be passed, which will be used to render the Wonderwall.
+     * The props should contain the template and variant.
+     * The props can optional contain the headline and ctaText.
+     *
+     * @param callback - The callback that should be called after the main functionality is done.
+     * The callback should be a function
+     *
+     */
+    renderWonderwall(container: HTMLElement, props: WonderwallProps, callback?: () => void): void;
```

Comment:
shouldn't we give it a name which is more integrator friendly, like renderLogin?

Reply:
renderAuthComponent(....) i guess we will use this method to display login or register

Reply:
but verification and recovery is also possible. But i think for now we only will use login and register, right?